assets/src/contexts/DeveloperInfoContext.tsx

```diff
@@ -17,12 +30,31 @@ export const DeveloperInfoProvider = ({ children }: Props) => {
   const [manualMode, setManualMode] = useState<boolean>(false);
   const [smartLayerSwitching, setSmartLayerSwitching] = useState<boolean>(false);
 
+  const [statisticsStatus, setStatisticStatus] = useState<boolean>(false);
+  const [statisticsData, setStatisticsData] = useState<Record<TrackIdentifier, Statistics>>({});
+  const updateStatisticsData = useCallback((id: TrackIdentifier, stats: VideoStats | AudioStats) => {
+    setStatisticsData((prev) => {
+      return {
+        ...prev, [id]: {
+          ...prev[id],
+          ...stats
+        }
+      };
+    });
+  }, [setStatisticsData]);
```

Comment:
```suggestion
  }, []);
```
The setter function should have stable reference.